### ISSUE TITLE
bug: Timer overlay window has same title as the main teamcraft window (breaks automation/accessibility)

### ISSUE THREAD

**Issue by rdj** [Supamiu/ffxiv-teamcraft#859]
**Describe the bug**
Automation and accessibility software cannot distinguish between the timer overlay and the main window because they have the same title, class, and exe.

Title: FFXIV Teamcraft
Class: Chrome_WidgetWin_1
Exe: ffxiv-teamcraft.exe

Just by changing the window title or class for the timer overlay, it could be targeted by automation / accessibility software separately from the main window. 

**To Reproduce**
1. Go to 'Alarms'
2. Click on 'Open Overlay' icon
3. Examine the window properties of the two windows, observing they are identical. The easiest way to do this is probably using Window Spy which comes with Autohotkey, but any number of automation / accessibility / developer programs can dump this information.

**Expected behavior**
The timer overlay window should have a different title (or class). 

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Desktop app
 - Version: v5.2.10

**Additional context**
I've never used Electron before, but it appears that this should be as simple as adding a `title` option when you create `BrowserWindow` from `openOverlay(url)` in `desktop/main.js`. Since the title of the overlay window isn't visible anyway, just using the URL or some artifact of the URL as the title would work fine. 

https://electronjs.org/docs/api/browser-window

If you're not inclined to make this change yourself but would accept a pull request just let me know and I can try to get teamcraft building on my machine.'

Thanks for your time and effort, teamcraft is an indispensable part of my ffxiv play experience! :)

**Comment by Supamiu** [comment]
Hey,

Thank you for the very detailed issue, I'm used to electron but didn't take time to properly configure everything so having some feedback is nice 👍 

Feel free to ping me on discord if you want more stuff or have more ideas regarding the desktop app or the app in general 😄 

This will be inside next patch, the one I'm going to release in an hour or so.

**Comment by rdj** [comment]
Hmmm, not fixed i v5.2.11. I reread the Electron documentation and it says that's the "default window title", since your window title was already FFXIV Teamcraft and you weren't setting that option before, I'm guessing that means the window title is getting updated based on the HTML `<title>` or something. A couple of things support this idea (emphasis mine): 

> Event: 'page-title-updated'
> Returns:
> 
> event Event
> title String
> Emitted when the document changed its title, calling event.preventDefault() will **prevent the native window's title from changing.**
>
> win.setTitle(title)
> title String
> Changes the title of native window to title.
> 
> win.getTitle()
> Returns String - The title of the native window.
> 
> Note: **The title of web page can be different from the title of the native window.**

So perhaps doing the recommended thing in a handler for `page-title-updated` would have the desired effect.

**Comment by Supamiu** [comment]
Oh, this means that I should implement SEO stuff, which was on the roadmap for 5.3 👍 

I'll do that but will be a bit later on, I would say during the next few weeks.

**Comment by rdj** [comment]
Verified fixed in v5.2.13. Thanks! 👍
